packages/js/src/core/revoke.test.ts

```diff
@@ -0,0 +1,24 @@
+import { revoke } from './revoke';
+
+describe('revoke', () => {
+  test('requester should called with correct parameters', async () => {
+    const revocationEndpoint = 'https://logto.dev/oidc/token/revocation';
+    const clientId = 'clientId';
+    const token = 'token';
+    const mockedRequester = jest.fn().mockResolvedValue({
+      ok: true,
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      json: async () => {},
```

Comment:
```suggestion
      json: () => Promise.resolve(),
```

Reply:
Replace optional parameter `fetchFucntion` with required `Requester`.
No longer need to mock it now.